CHANGELOG.md

```diff
@@ -1,5 +1,34 @@
 ## UNRELEASED
 
+Improvements:
+
+* Support Consul namespaces [Enterprise feature] in all consul-k8s components
+* Create allow and deny lists of k8s namespaces for catalog sync and Connect inject
+* Significantly increases the test coverage of all components
+
+Bug Fixes:
+
+* Bootstrap ACLs: Allow users to update their Connect ACL binding rule definition on upgrade
+* Bootstrap ACLs: Fixes mesh gateway ACL policies to have the correct permissions
+* Sync: Fixes a hot loop bug when getting an error from Consul when retrieving service information
+
+Notes:
+
+* Sync: The underlying call to Consul to retrieve services has been switched to do so by the synthetic node. This causes a slight behavior change in that we will no longer remove services with the `ConsulK8STag` if it's not attached to the `k8s-sync` node.
+
+* Sync: The allow/deny list setting change is backwards compatible for syncing all namespaces, however if if a user upgrades consul-k8s without upgrading the Helm chart as well, there will be a slight difference in behavior for syncing all namespaces in that it won't automatically exclude `kube-system` on its own.
+
+* Sync: Two sync processes running from separate namespaces in the same k8s cluster will now require different tags or they will interfere will each other's service registrations.
+
+* Bootstrap ACLs: Previously, ACL policies were not updated after creation. Now, if namespaces are enabled, they are updated every time the ACL bootstrapper is run so that any namespace config changes can be adjusted. This is only an issue if you are manually updating ACL policies after creation.
```

Comment:
We should clarify what "This" refers to in the last sentence. I also think we shouldn't call it a manual update bacause someone might have written API automation specific to their use case.